### PR TITLE
Malloy CLI plugin

### PR DESCRIPTION
**Pull request by Ben8t**

### CODE REVIEW

File: plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    )
+})
+public class CLI extends AbstractExecScript {
+    @Schema(
+        title = "Docker options when using the `DOCKER` runner"
+    )
+    @PluginProperty
+    @Builder.Default
+    protected DockerOptions docker = DockerOptions.builder()
+        .image("malloy")
```
Comment (Ben8t) [inline plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java:70]: my only concern is about the Docker image 🤔 

I make this works with the following `Dockerfile` locally (with pullPolicy: NEVER)
```
FROM node:latest

WORKDIR /usr/app

RUN npm install duckdb && \
    npm install -g malloy-cli
   ```
   
   and building with `docker build -t malloy . --platform=linux/amd64`
   
I maybe need some help on this to build a proper Docker image that we can host [here](https://github.com/orgs/kestra-io/packages) (while waiting for a proper image coming from Malloy's team)

File: plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    )
+})
+public class CLI extends AbstractExecScript {
+    @Schema(
+        title = "Docker options when using the `DOCKER` runner"
+    )
+    @PluginProperty
+    @Builder.Default
+    protected DockerOptions docker = DockerOptions.builder()
+        .image("ghcr.io/kestra-io/malloy")
```
Comment (Ben8t) [inline plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java:70]: my only concern is about the Docker image 🤔 

I make this works with the following Dockerfile(https://github.com/kestra-io/examples/blob/main/dockerfiles/malloy.Dockerfile) and build as `ghcr.io/kestra-io/malloy:latest` - see [here](https://github.com/kestra-io/examples/pkgs/container/malloy)

I have a doubt about the `--platform=linux/amd64`

Comment (anna-geller) [reply plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java:67]: do you mean without it it fails on your mac or in CI?

File: plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                           }
+               
+                     - id: run_malloy
+                       type: io.kestra.plugin.scripts.malloy.CLI
+                       commands:
+                         - malloy-cli run model.malloy
+                """
+    )
+})
+public class CLI extends AbstractExecScript {
```
Comment (tchiotludo) [inline plugin-script-malloy/src/main/java/io/kestra/plugin/scripts/malloy/CLI.java:60]:
```suggestion
public class Cli extends AbstractExecScript {
```
No upper case please

**Review by tchiotludo — CHANGES_REQUESTED** [review]

**Comment by tchiotludo** [comment]
move to https://github.com/kestra-io/plugin-malloy

**Pull request closed without merging**